core/src/main/java/org/ehcache/config/writebehind/DefaultWriteBehindConfiguration.java

```diff
@@ -28,15 +28,14 @@
 public class DefaultWriteBehindConfiguration implements WriteBehindConfiguration {
 
   private int minWriteDelay = 0;
-  private int maxWriteDelay = 1;
-  private int rateLimitPerSecond = 0;
+  private int maxWriteDelay = Integer.MAX_VALUE;
+  private int rateLimitPerSecond = Integer.MAX_VALUE;
   private boolean writeCoalescing = false;
-  private boolean writeBatching = false;
   private int writeBatchSize = 1;
-  private int retryAttempts = 1;
+  private int retryAttempts = 0;
   private int retryAttemptDelaySeconds = 1;
```

Comment:
No default, and no error in the place of default zero retry attempts.  If you specify a non-zero retry attempts value then you must also specify the delay you want: time to break out the pink library.
![Schematron!](https://bitbucket-assetroot.s3.amazonaws.com/c/photos/2014/Apr/23/m-xsl-schematron-transformer-logo-3308696766-2_avatar.png)